pyspecdata/core.py

```diff
@@ -69,7 +69,6 @@
 import matplotlib
```

Comment:
@wfkincaid the previous PR was a complicated situation b/c we apparently did those fixes on a version of core where black had not been run.  Here, I fixed that and properly merged in master.

Don't forget to look at the diff -- before you could see the added conflict makers.

Reply:
please review this guy and comment that you've done so, so that I can merge it